doc/introduction/interfaces/tf.rst

```diff
@@ -5,10 +5,10 @@ TensorFlow interface
 
 In order to use PennyLane in combination with TensorFlow, we have to generate TensorFlow-compatible
 quantum nodes. A basic ``QNode`` can be translated into a quantum node that interfaces with TensorFlow,
-either by using the ``interface='tf'`` flag in the QNode Decorator, or by calling the
-:meth:`QNode.to_tf() <pennylane.qnodes.JacobianQNode.to_tf>` method. Internally, the translation is
-executed by the :func:`~.interfaces.tf.to_tf` function that returns the
-new quantum node object.
+either by using the ``interface='tf'`` flag in the QNode decorator or QNode class constructor, or by
+calling the :meth:`QNode.to_tf() <pennylane.qnodes.JacobianQNode.to_tf>` method. Internally, the
+translation is executed by the :func:`~.interfaces.torch.to_torch` function that dynamically modifies
```

Comment:
I don't think we need to mention internal details here.

As a note, the interfaces are mix-in classes that can get added onto a tape by `~.interfaces.tf.TFInterface.apply`, but even most developers are not going to be using that information.